tests/cli/workflow/conftest.py

```diff
@@ -34,13 +34,13 @@ def analysis_store(base_store: Store, workflow_case_id: str, helpers: StoreHelpe
     case = helpers.add_case(_store, workflow_case_id, data_analysis=Pipeline.MIP_DNA)
 
     dna_sample = helpers.add_sample(
-        _store, "dna_sample", is_rna=False, last_sequenced_at=datetime.now(), reads=10000000
+        _store, "dna_sample", is_rna=False, reads=10000000, last_sequenced_at=datetime.now()
```

Comment:
why do we reshuffle so many parameters?

Reply:
I guess creating test cases. Should be refactored in another PR./